echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/PipelineConfigsPollingAgent.groovy

```diff
@@ -79,7 +79,7 @@ class PipelineConfigsPollingAgent extends AbstractPollingAgent {
       log.info("Running the pipeline configs polling agent...")
 
       // Only interested in pipelines that have triggers and that too scheduled triggers
-      def pipelines = pipelineCache.getPipelines().toBlocking().first().findAll {
+      def pipelines = pipelineCache.getPipelines().findAll {
```

Comment:
Given that this blocked before, should we be using `getPipelinesSync` here?  (This looks like it will throw an NPE if called before the first polling cycle.)

Reply:
Yes indeed, good catch